### DISCUSSION TITLE
Using vedo to real time visualisation.

### DISCUSSION THREAD

**Discussion by IgorZaton** [marcomusy/vedo#346] [Q&A] [answered]
Hi,
I'm trying to make stereo vision system that could work in the real time. Is it possible to update point cloud without closing the window so it could work continuously?

**Comment by marcomusy** [comment, marked as answer]
Hi, you can reset/modify point positions with 
```python
mypoints = vedo.Points(some_points)
show(mypoints)
mypoints.points(new_points_coords)
show(mypoints)
```

if instead you need to change the nr of points you need to create a new `Points` object.

**Reply by IgorZaton** [reply]
Yes, but when I load the point cloud like this:
```python
pts = vedo.Points(some_points)
while(True):
    vedo.show(pts)
    # generating some new points
    pts.points(new_points)
```
Window kind of freeze everything and next point cloud is shown when I close the current window. After doing that, pops up new window with new data. Is there a way to update points in one window in real time so it would work like animation?

**Reply by marcomusy** [reply]
use:
`vedo.show(pts, interactive=False)`

**Reply by IgorZaton** [reply]
It's working! Thanks for help and quick response :D
